### ISSUE TITLE
(TBD) Unified coding style rule

### ISSUE THREAD

**Issue by wysohn** [TriggerReactor/TriggerReactor#568]
https://editorconfig.org/

It seems like both IntelliJ and VSCode have EditorConfig plugins, so it should be a good start

But also open for other suggestions

**Comment by Cupelt** [comment]
Do you mean TRG official coding style rule?

**Comment by wysohn** [comment]
> Do you mean TRG official coding style rule?

Yeah I usually just use auto format (ctrl alt L), and I think most of us do, but it can differ slightly depending on the IDE specific settings

Having an rule, we can just let IDE to adjust formatting according to the rule, and we will be all using the same format without much hassle

**Comment by Cupelt** [comment]
It looks good!
